pkgs/build-support/php/build-php.nix

```diff
@@ -0,0 +1,93 @@
+{ stdenvNoCC, lib, writeTextDir, composer, php }@inputs:
+
+{ pname
+, version
+, buildInputs ? [ ]
+, nativeBuildInputs ? [ ]
+, src
+, vendorOutputHash
```

Comment:
maybe rename to `vendorSha256` similar to go build support

Reply:
I don't know if it's a good idea to have the suffix `Sha256`. Since a hash can be made from different encryption algorithm, I believe it's better to leave it as it is or eventually use `vendorHash` ?

Reply:
So, `vendorOutputHash` or `vendorHash` ? :)

Reply:
`vendorHash` sounds good to me

Reply:
Done.